.github/workflows/continuous-integration-workflow.yml

```diff
@@ -0,0 +1,72 @@
+env:
+  RUBY_VERSION: 2.6.x
+  POSTGRES_USER: postgres
+  POSTGRES_PASSWORD: postgres
+  POSTGRES_DB: safe_pg_migrations_test
+  PGPASSWORD: postgres
+
+name: Rake tests
+on: pull_request
+
+jobs:
+  rubocop-test:
+    name: Rubocop
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: doctolib/checkout@v1
+      - uses: doctolib/setup-ruby@v1
+        with:
+          ruby-version: ${{ env.RUBY_VERSION }}
+      - uses: doctolib/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
+      - name: bundle
+        run: |
+          gem install bundler
+          bundle config path vendor/bundle
+          bundle install --jobs 4 --retry 3
```

Comment:
I've tried to move this part here in an independent action, but as document in https://github.com/actions/runner/issues/438 it is not yet possible to use other actions in composite actions.

Apparently github is working on it, once done we could have an internal action in a repo that we can use in any of our repo.

Reply:
That would also make the file a lot shorter :)